packages/shared/src/components/sidebar/ContributeSection.tsx

```diff
@@ -0,0 +1,86 @@
+import dynamic from 'next/dynamic';
+import React, { ReactElement, useContext, useState } from 'react';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import EmbedIcon from '../icons/Embed';
+import LinkIcon from '../icons/Link';
+import { ListIcon, SidebarMenuItem } from './common';
+import { Section, SectionCommonProps } from './Section';
+
+const SubmitArticleModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "SubmitArticleModal" */ '../modals/SubmitArticleModal'
+    ),
+);
+
+const NewSourceModal = dynamic(
+  () =>
+    import(/* webpackChunkName: "NewSourceModal" */ '../modals/NewSourceModal'),
+);
+
+export function ContributeSection(props: SectionCommonProps): ReactElement {
+  const {
+    canSubmitArticle,
+    submitArticleOn,
+    submitArticleSidebarButton,
+    submitArticleModalButton,
+  } = useContext(FeaturesContext);
```

Comment:
I think we can pass these down from the Sidebar as the context is already loaded there, what do you think?

Reply:
Was thinking this is one too many variables to pass, and just to keep the props slim, also as much as to take away from the `Sidebar`, but we can definitely do that.

Reply:
Was thinking this is one too many variables to pass, and just to keep the props slim, also as much as to take away from the Sidebar, but we can definitely do that. WDYT?

Reply:
I think anything outside of analyticsContext makes sense it will make the components more dynamic as well.
But eventually leaving it up to you to decide 🙏

Reply:
Thanks for the thoughts. Let me apply some refactoring on this 🚢